packages/backend/src/shared/baseModel.test.ts

```diff
@@ -18,9 +18,6 @@ export const getPageTests = <Type extends BaseModel>({
       for (let i = 0; i < 22; i++) {
         modelsCreated.push(await createModel())
       }
-      if (sortOrder === SortOrder.Desc) {
```

Comment:
Line 13 still picks a sort order and random, doesn't it?

Reply:
I had moved this logic down but I actually think Max wanted it out as well. So I've removed here.